tests/returntypes/test_jax_qnode_new.py

```diff
@@ -1223,7 +1215,7 @@ def circuit(n, a):
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
 
-# TODO: Add "jax-jit"
+# TODO: add support for fwd mode to JAX-JIT
```

Comment:
Is this comment still relevant if the JIT tests are in a dedicated file?

Reply:
Good catch, removed. Thanks!